puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_develop.sh

```diff
@@ -25,7 +25,8 @@ while ! bundle install --without=development -j 5; do
 done
 
 # run npm install if package.json exists and we are running integration tests (which only run on postgresql)
-[ ${database} = postgresql ] && [ -e "$APP_ROOT/package.json" ] && npm install
+# and upgrade npm to the newest version first
+[ ${database} = postgresql ] && [ -e "$APP_ROOT/package.json" ] && npm install npm && $APP_ROOT/node_modules/.bin/npm install
```

Comment:
Nitpick, can you rewrite this into an `if` block please? (`if [ ${database} = postgresql -a -e "$APP_ROOT/package.json" ]; then` with the two commands within the block? More than one command in a single line gets difficult to follow.